drivers/gpio/Kconfig

```diff
@@ -1407,6 +1408,14 @@ config GPIO_WM8994
 	  Say yes here to access the GPIO signals of WM8994 audio hub
 	  CODECs from Wolfson Microelectronics.
 
+config GPIO_TS7800_V2
```

Comment:
I'd argue this should be "GPIO_TS7800V2" for consistency since #defines in the code use TS7800V2 and not TS7800_v2